frontend/src/routes/order/updateOrderStatus/+page.svelte

```diff
@@ -1,21 +1,30 @@
 <script lang="ts">
-	let statusList = ['making', 'Arriving', 'delivering'];
+	import Deliver from '$lib/assets/deliver.svg';
+	import Accept from '$lib/assets/accept.svg';
+	import Arrival from '$lib/assets/arrival.svg';
+	import Making from '$lib/assets/making.svg';
+	import OrderStatusCard from '$lib/components/PUA/orderStatusCard.svelte';
+
+	let statusCardContent: {
+		time: string;
+		price: string;
+		src: string;
+		text: string;
+		user: string;
+	}[] = [
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Making, text: 'Making', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Deliver, text: 'Deliver', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Accept, text: 'Accept', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Arrival, text: 'Arrival', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Accept, text: 'Accept', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Making, text: 'Making', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Deliver, text: 'Deliver', user: 'user01' },
+		{ time: '2020-10-11 19:20:50', price: '99999', src: Arrival, text: 'Arrival', user: 'user01' }
+	];
 </script>
 
-<div class="flex flex-wrap items-center gap-4 p-10">
-	{#each statusList as status}
-		<div class=" flex items-center rounded-xl bg-neutral-300 p-4">
-			<div class="w-64">
-				<div class=" text-xl font-bold leading-relaxed text-PUA-dark-orange">2020</div>
-				<div class=" text-xl font-bold leading-relaxed text-PUA-dark-orange">NT9999</div>
-				<div class=" text-xl font-bold leading-relaxed text-PUA-dark-orange">user ATM</div>
-			</div>
-			<div>
-				<button
-					class="h-20 w-20 rounded-lg bg-PUA-stone p-2 text-center font-bold leading-relaxed text-white"
-					>{status}</button
-				>
-			</div>
-		</div>
+<div class=" flex flex-wrap items-center justify-center gap-10 bg-gray-200 py-12">
```

Comment:
use min-h-screen to prevent
![image](https://github.com/PUArallelepiped/PUN-street-Universal-Access/assets/93046487/02e7c5b2-efe3-4fbb-a6df-1699159a2bcf)

Reply:
直接進去網址就會變白色，先進到root再進去 localhost:5173/order/updateOrderStatus就不會